share/spice/rand_word/rand_word.js

```diff
@@ -34,23 +31,38 @@
                 }
             }
         };
-        
-        if ( list_of_list[0].length == 1 ) {
-            spiceObj.data = {
-                title: "Random Word",
-                word: first.word
-            };
-            
-            spiceObj.templates = {
-                group: 'text',
-                options: {
-                    content: Spice.rand_word.single_word,
-                    moreAt: true
+        if (list_of_list[0].length == 1) {
+            get_definition(first.word).done(function(api_result) {
```

Comment:
the `get_definition` function doesn't seem to serve much purpose. Let's just use the canonical form:

```js
$.getJSON("/js/spice/rand_word_fetch_id/" + word, function(data) {
    var first = data[0];
    $.extend(spiceObj, {
        data: { .... },
        templates: { .... },
        normalize: funciton() { ... }
    });

    Spice.add(spiceObj);
}
```